From cf996872161bdc05293aaf8b82e57f68542eb055 Mon Sep 17 00:00:00 2001
From: FrancescoFerraro <francesco.f@variscite.com>
Date: Thu, 17 Nov 2022 07:45:27 +0100
Subject: [PATCH 12/12] imx8mm_var_dart: disable OPTEE lib by default

This change avoids the following error when linux boots without
tee.bin in the U-Boot image (default condition).

optee: probe of firmware:optee failed with error -22

Signed-off-by: FrancescoFerraro <francesco.f@variscite.com>
(cherry picked from commit aed2ad851e01ce706d82a941d47c4205da51ef3e)
Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>
---
 configs/imx8mm_var_dart_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/imx8mm_var_dart_defconfig b/configs/imx8mm_var_dart_defconfig
index ba3a69ae569..599eaf5c4ed 100644
--- a/configs/imx8mm_var_dart_defconfig
+++ b/configs/imx8mm_var_dart_defconfig
@@ -153,3 +153,4 @@ CONFIG_EFI_CAPSULE_ON_DISK=y
 CONFIG_EFI_IGNORE_OSINDICATIONS=y
 CONFIG_EFI_CAPSULE_FIRMWARE_RAW=y
 CONFIG_EFI_SECURE_BOOT=y
+# CONFIG_OPTEE_LIB is not set
-- 
2.34.1

